CAVEATS.md

```diff
@@ -0,0 +1,52 @@
+# Platform specific notes
+
+## Windows
+
+### Spectest requirements
+
+You need recent version of either Strawberry Perl or ActiveState Perl.
+
+If you are working with ActiveState Perl you need the Mingw gcc compiler.
+
+You need msys git installed and you need "\Program Files\Git\cmd" on your
+execution path and NOT "\Program Files\Git\bin".
+
+You need a win32 curl program.
+
+
+## macOS
+
+### Dynamic libraries
+
+If Rakudo is installed into $HOME directory (this is also the case when
+rakubrew <https://rakubrew.org/> is used) then some modules may fail to
+install due to missing dynamic libraries. This could be caused by a
+situation when the library is not provided by macOS system itself but
+installed using Homebrew or MacPorts package managers. In this case the
+library cannot be loaded by rakudo binary. This is a security precaution
```

Comment:
*the* rakudo binary. Also, will you capitalize Rakudo?

Reply:
It is a file name here.